src/analytics/Properties.tsx

```diff
@@ -1326,6 +1326,13 @@ interface SwapEventsProperties {
     tokenSymbol: string
     tokenId: string
     tokenNetworkId: string
+    fromTokenSymbol: string | undefined
+    fromTokenId: string | undefined
+    fromTokenNetworkId: string | undefined
+    toTokenSymbol: string | undefined
+    toTokenId: string | undefined
+    toTokenNetworkId: string | undefined
+    switchedNetworkId: boolean
```

Comment:
Adding from and to here though it will duplicate some of the existing info, but will give easier access to what cross chain pairs users are interested in.